clusterman/kubernetes/kubernetes_cluster_connector.py

```diff
@@ -265,6 +256,29 @@ def _count_batch_tasks(self, node_ip: str) -> int:
                     break
         return count
 
+    def _is_pod_scheduled(self, pod: KubernetesPod) -> bool:
```

Comment:
this name worries me a tiny bit as i could see myself (or a future yelper) thinking that this could return `True` for a Pod that was scheduled and is currently running - should this also return `True` in that case?

Reply:
Yes, You are right, I named it "_is_pod_pending_scheduled" but I didn't like so much. Let me think again